### PR TITLE
Add new command for new path

### PR DESCRIPTION
**Pull request by tanrax**

### CODE REVIEW

File: bin/shipit [hunk 1]
```diff
@@ -45,6 +45,7 @@ usage() {
 	echo "    console      shell, ssh  Open an SSH session on the remote host"
 	echo "    exec <cmd>   run         Execute <cmd> on the remote host"
 	echo "    copy <file>  cp          Copy <files> to the remote host"
+	echo "    --config     -c          Path config file"
```
Comment (sapegin) [inline bin/shipit]: “Config file path (default: .shipit)”.

File: bin/shipit [hunk 2]
```diff
@@ -239,6 +240,7 @@ done
 arg=$1; shift
 if [ -n "$arg" ]; then
 	case $arg in
+		-c|--config) CONFIG_NAME="$@";;
```
Comment (sapegin) [inline bin/shipit]: I believe it should be in the options loop above.

File: bin/shipit (hunk 1 above)
Comment (sapegin) [reply bin/shipit]: It’s actually a file name, not a path.

Comment (tanrax) [reply bin/shipit]: Hi! Thanks to you for the fantastic script. I use it every day 👍 
Excuse my English, is that correct now?

File: bin/shipit (hunk 2 above)
Comment (tanrax) [reply bin/shipit]: I don't understand where you mean. Where did I put it now?

File: bin/shipit (hunk 1 above)
Comment (sapegin) [reply bin/shipit]: Almost ;-) Config file path → Config file name.

File: bin/shipit (hunk 2 above)
Comment (sapegin) [reply bin/shipit]: Here: 

https://github.com/sapegin/shipit/blob/5e6e2eeae5aff0000936b6c8e2bcd4f7583205a1/bin/shipit#L231-L236

**Comment by tanrax** [comment]
I can't get it to work as an option.

**Comment by sapegin** [comment]
I think you need to call `shift` twice: to remove `--config` and to remove the value.

**Comment by tanrax** [comment]
Already working!

**Comment by sapegin** [comment]
Awesome, thanks!

**Comment by tanrax** [comment]
To you for guiding me and teaching me. Fantastic script!

**Pull request merged**